gems/sorbet-runtime/lib/types/configuration.rb

```diff
@@ -131,6 +131,11 @@ def self.default_checked_level=(default_checked_level)
   # Parameters passed to value.call:
   #
   # @param [TypeError] error TypeError that was raised
+  # @param [Hash] opts A hash containing contextual information on the error:
+  # @option opts [String] :kind One of:
+  #   ['T.cast', 'T.let', 'T.bind', 'T.assert_type!', 'T.must', 'T.absurd']
+  # @option opts [Object, nil] :type Expected param/return value type
+  # @option opts [Object] :value Actual param/return value
   #
   # @example
   #   T::Configuration.inline_type_error_handler = lambda do |error|
```

Comment:
did you want to update this example to mention / use `opts`?

Reply:
I did, thanks for catching!